<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundDialog</class>
 <widget class="QDialog" name="SoundDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play Sound</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="soundList"/>
    </item>
    <item>
     <widget class="QToolButton" name="playButton">
      <property name="toolTip">
       <string>Play Sound Locally</string>
      </property>
      <property name="statusTip">
       <string>Play Sound Locally</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/playbutton.png</normaloff>:/playbutton.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sendButton">
      <property name="toolTip">
       <string>Play Sound to Clients</string>
      </property>
      <property name="statusTip">
       <string>Play Sound to Clients</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/sendbutton.png</normaloff>:/sendbutton.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="stopButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/stopbutton.png</normaloff>:/stopbutton.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="mp3SoundList"/>
    </item>
    <item>
     <widget class="QToolButton" name="mp3PlayButton">
      <property name="toolTip">
       <string>Play Sound Locally</string>
      </property>
      <property name="statusTip">
       <string>Play Sound Locally</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/playbutton.png</normaloff>:/playbutton.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="mp3SendButton">
      <property name="toolTip">
       <string>Play Sound to Clients</string>
      </property>
      <property name="statusTip">
       <string>Play Sound to Clients</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/sendbutton.png</normaloff>:/sendbutton.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="mp3StopButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/stopbutton.png</normaloff>:/stopbutton.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SoundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SoundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
